curriculumcourse/pom.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2017 Red Hat, Inc. and/or its affiliates.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~       http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<!--
+  ~ The file is modified by XSL transformation before kie-wb-playground.zip file is created.
+  ~
+  ~ The following modifications are applied:
+  ~ * Remove <parent> element
+  ~ * Remove <dependencies> element
+  ~ * Add <groupId> element corresponding to current value of <artifactId> element
+  ~ * Add <version> element with value 1.0.0-SNAPSHOT
+  -->
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.kie.workbench.playground</groupId>
+    <artifactId>playground-parent</artifactId>
+    <version>7.0.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>curriculumcourse</artifactId>
+  <packaging>kjar</packaging>
+  <name>Course Scheduling Example</name>
```

Comment:
I am not sure if the name is unique enough.
If the user manual, slides or our mails talk about "look at the Course Scheduling example", that's going to be ambiguous and therefore cause misunderstandings, because we have 2 examples with that name: one in optaplanner-examples and one in kie-wb-playground.
For cloudbalancing we've avoided that by calling the wb one "optacloud".

We need to figure out a good naming convention to distinguish between the optaplanner-examples and the kie-wb-playground optaplanner-wb examples consistently.